src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
```

Comment:
[CODE STYLE] - Enum constant names usually start with capital letter. Please revise the `Shape` enum to follow this convention, for example, `Triangle` should be `TRIANGLE`, `Circle` should be `CIRCLE`, and `Rectangle` should be `RECTANGLE`.